candi/cloud-providers/aws/openapi/cluster_configuration.yaml

```diff
@@ -33,6 +35,7 @@ apiVersions:
               The number of master nodes to create.
           instanceClass:
             type: object
+            default: {}
```

Comment:
Why we need this default ?

Reply:
These defaults are not needed, I forgot to remove them after the tests.